sys/riscv/riscv/pmap.c

```diff
@@ -2013,6 +2013,18 @@ pmap_remove_kernel_l2(pmap_t pmap, pt_entry_t *l2, vm_offset_t va)
 	    __func__, l2, oldl2));
 }
 
+static inline void
```

Comment:
```suggestion
static __inline void
```
in this file it seems